pkgs/development/tools/gomod2nix/src.json

```diff
@@ -0,0 +1 @@
+{"owner": "tweag", "repo": "gomod2nix", "sha256": "0bgr7h1clfyfl5l25h65nhcw0d0drf77zw0v1p5fjdymiqb0yk8j", "rev": "v1.4.0"}
```

Comment:
I am not really a fan of extracting such basic data about a package into a json file. Leaving that inside the package would be my preference. We also have good tooling already to update those information like nix-update. Accessing the attributes from fetchFromGitHub should also be possible otherwise I'd rather passthru more there.